versions.properties

```diff
@@ -0,0 +1,41 @@
+## suppress inspection "SpellCheckingInspection" for whole file
```

Comment:
👀

Reply:
This is from `refreshVersions` plugin, part of https://github.com/duckduckgo/Android/pull/1090